openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_objecttag_export_helpers.py

```diff
@@ -0,0 +1,189 @@
+        # Tag blocks
+        sequential_tags1 = api.tag_content_object(
+            object_key=self.sequential.location,
+            taxonomy=self.taxonomy_1,
+            tags=['Tag 1.1', 'Tag 1.2'],
+        )
+        sequential_tags2 = api.tag_content_object(
+            object_key=self.sequential.location,
+            taxonomy=self.taxonomy_2,
+            tags=['Tag 2.1'],
+        )
+        xblock = self.store.get_item(self.sequential.location)
+        tagged_sequential = TaggedContent(
+            display_name=xblock.display_name_with_default,
+            block_id=str(xblock.location),
+            category=xblock.category,
+            children=[],
+            object_tags={
+                self.taxonomy_1.id: list(sequential_tags1),
+                self.taxonomy_2.id: list(sequential_tags2),
+            },
+        )
+
+        assert self.expected_tagged_xblock.children is not None  # type guard
+        self.expected_tagged_xblock.children.append(tagged_sequential)
+
+        vertical = self.store.create_child(self.user_id, self.sequential.location, "vertical", "test_vertical1")
+        vertical_tags = api.tag_content_object(
+            object_key=vertical.location,
+            taxonomy=self.taxonomy_2,
+            tags=['Tag 2.2'],
+        )
+        xblock = self.store.get_item(vertical.location)
+        tagged_vertical = TaggedContent(
+            display_name=xblock.display_name_with_default,
+            block_id=str(xblock.location),
+            category=xblock.category,
+            children=[],
+            object_tags={
+                self.taxonomy_2.id: list(vertical_tags),
+            },
+        )
+
+        assert tagged_sequential.children is not None  # type guard
+        tagged_sequential.children.append(tagged_vertical)
+
+        vertical2 = self.store.create_child(self.user_id, self.sequential.location, "vertical", "test_vertical2")
+        xblock = self.store.get_item(vertical2.location)
+        tagged_vertical2 = TaggedContent(
+            display_name=xblock.display_name_with_default,
+            block_id=str(xblock.location),
+            category=xblock.category,
+            children=[],
+            object_tags={},
+        )
+        assert tagged_sequential.children is not None  # type guard
+        tagged_sequential.children.append(tagged_vertical2)
+
+        html = self.store.create_child(self.user_id, vertical2.location, "html", "test_html")
+        html_tags = api.tag_content_object(
+            object_key=html.location,
+            taxonomy=self.taxonomy_2,
+            tags=['Tag 2.1'],
+        )
+        xblock = self.store.get_item(html.location)
+        tagged_text = TaggedContent(
+            display_name=xblock.display_name_with_default,
+            block_id=str(xblock.location),
+            category=xblock.category,
+            children=[],
+            object_tags={
+                self.taxonomy_2.id: list(html_tags),
+            },
+        )
+
```

Comment:
Could you also add some "deleted" object tags to ensure they are omitted from the results here?
cf https://github.com/open-craft/edx-platform/pull/622/commits/35a3d2b187fc583dccaf4f6f7185b07745409964